src/models/relations.js

```diff
@@ -22,21 +24,29 @@ limitations under the License.
  * The typical way to get one of these containers is via
  * EventTimelineSet#getRelationsForEvent.
  */
-export default class Relations {
+export default class Relations extends EventEmitter {
     /**
      * @param {String} relationType
      * The type of relation involved, such as "m.annotation", "m.reference",
      * "m.replace", etc.
      * @param {String} eventType
      * The relation event's type, such as "m.reaction", etc.
+     * @param {?Room} room
```

Comment:
isn't it usually `{Room?}` ?

Reply:
I don't use JSDoc type syntax much, but that seems to the nullable type convention there: https://jsdoc.app/tags-type.html